we.go

```diff
@@ -30,7 +30,7 @@ type configuration struct {
 type cond struct {
 	ChanceOfRain   string  `json:"chanceofrain"`
 	FeelsLikeC     int     `json:",string"`
-	PrecipMM       float32 `json:"precipMM,string"`
+	PrecipMM       int     `json:"precipMM,string"`
```

Comment:
The type-change float32->int leads to errors like: `json: cannot unmarshal number 0.0 into Go value of type int`